build.sbt

```diff
@@ -257,16 +257,14 @@ lazy val scalameta = crossProject
 lazy val scalametaJVM = scalameta.jvm
 lazy val scalametaJS = scalameta.js
 
-lazy val semanticdbScalac = project
-  .in(file("scalameta/semanticdb-scalac"))
+lazy val semanticdbScalacPlugin = project
+  .in(file("scalameta/semanticdb-scalac-plugin"))
   .settings(
     moduleName := "semanticdb-scalac",
-    description := "Scala 2.x compiler plugin that generates semanticdb on compile",
+    description := "Fatjar of semanticdbScalac",
```

Comment:
I suggest we keep an old description here.

Reply:
Done.